lang/ruby34/Portfile

```diff
@@ -0,0 +1,229 @@
+# unconditional.  Nevertheless, we limit it to the failing case.
+#
+if {[string match *clang* ${configure.compiler}] \
+    && ${configure.cxx_stdlib} eq "libstdc++"} {
+                    configure.cxxflags-append -std=c++03
+}
+
+configure.args      --enable-shared \
+                    --enable-install-static-library \
+                    --disable-install-doc \
+                    --mandir="${prefix}/share/man" \
+                    --enable-pthread \
+                    --disable-yjit \
+                    --disable-rjit \
+                    --without-gmp \
+                    --enable-mkmf-verbose \
+                    --with-opt-dir="${prefix}" \
+                    --program-suffix=${ruby_ver} \
+                    --with-rubyhdrdir=${prefix}/include/ruby-${version} \
+                    --with-rubylibprefix="${prefix}/lib/ruby${ruby_ver}" \
+                    --with-openssl-dir=[openssl::install_area] \
+                    --without-baseruby
+# prefer Apple cctools to GNU binutils, build with binutils may fail.
+configure.args-append   AR=${prefix}/bin/ar RANLIB=${prefix}/bin/ranlib
+
+# Add the architecture flag as required
+if {[info exists build_arch] && ${build_arch} != ""} {
+    configure.args-append "--with-arch=${build_arch}"
+}
+
+post-patch {
+    # Ensure that generated files have clearly later mtimes than sources,
+    # to avoid gratuitous rebuilds by make.
+    # This is already nominally true based on the patching order, but
+    # one-second resolution on timestamps may cover it up.
+    #
+    # The list of files here should match the list of files patched by
+    # patch-generated.diff.
+    #
+    # Increment the mtime(s) by one second.
+    foreach genfile [list configure] {
+        set genfull ${worksrcpath}/${genfile}
+        file mtime ${genfull} [expr [file mtime ${genfull}] + 1]
+    }
+}
+
+post-destroot {
+    foreach type {site vendor} {
+        set libdir ${destroot}${prefix}/lib/ruby${ruby_ver}/${type}_ruby/${version}
+        xinstall -m 0755 -d ${libdir}
+        foreach subdir [exec find ${libdir} -type d -empty] {
+            destroot.keepdirs-append ${subdir}
+        }
+    }
+
+    # install destination of commands from port:rb34*
+    xinstall -m 0755 -d ${destroot}${prefix}/libexec/ruby${ruby_ver}
+    destroot.keepdirs-append ${destroot}${prefix}/libexec/ruby${ruby_ver}
+}
+
+notes-append "
+To make this the default Ruby (i.e., the version run by\
+the 'ruby', 'gem' or 'bundle' commands), run:
+    sudo port select --set ruby $name
+"
+
+variant doc description "Install rdoc indexes and C API documents" {
+        configure.args-delete   --disable-install-doc
+}
+
+variant gmp description "use gmp" {
+        configure.args-delete   --without-gmp
+        depends_lib-append      port:gmp
+}
+
+variant jemalloc description "use jemalloc" {
+        configure.args-delete   --without-jemalloc
+        configure.args-append   --with-jemalloc
```

Comment:
please use `configure.args-replace` here and elsewhere in the Portfile

Reply:
OK.  This was just copied from `ruby33`, which was copied from `ruby32`, etc.

Reply:
Fixed now.